tests/Unit/Utilities/Test_Functional.cpp

```diff
@@ -21,9 +21,28 @@
 #include "Utilities/TypeTraits.hpp"
 #include "tests/Utilities/MakeWithRandomValues.hpp"
 
+// IWYU pragma: no_include <algorithm>
+
 namespace funcl {
 using std::imag;
 using std::real;
+using std::max;
+using std::min;
+// Need to provide overloads of min/max for double with float because of the
+// tensor product of typelists in the tests.
```

Comment:
Why are these only in the test? So this won‘t work outside the test?

Reply:
It shouldn't work outside the test: https://en.cppreference.com/w/cpp/algorithm/min

The reason they're in the test is so I don't have to write a bunch of extra code to get a test working with only `double-double` and `float-float`

Reply:
Could you add this to the comment please?

Reply:
Yes, of course :)

Reply:
This is done now. Let me know if you're happy with the comment :)